src/sentry/static/sentry/app/components/events/errors.tsx

```diff
@@ -54,6 +57,35 @@ class EventErrors extends React.Component<Props, State> {
     }
   }
 
+  async fetchReleaseArtifacts(query: string) {
+    const {api, orgSlug, event, projectSlug} = this.props;
+    const {release} = event;
+    const releaseVersion = release?.version;
+
+    if (!releaseVersion || !query) {
+      return;
+    }
+
+    try {
+      const releaseArtifacts = await api.requestPromise(
+        `/projects/${orgSlug}/${projectSlug}/releases/${encodeURIComponent(
+          releaseVersion
+        )}/files/?query=${query}`,
+        {
+          method: 'GET',
+        }
+      );
+
+      this.setState({releaseArtifacts});
+    } catch (error) {
+      Sentry.withScope(scope => {
+        scope.setLevel(Sentry.Severity.Error);
```

Comment:
The level should be error by default given you're calling `captureException`.
This level trick is needed if you're doing a `captureMessage`.

Reply:
also, every following event would be level=`Error` as the Scope's level has precedence over Event's level.

Reply:
@marandaneto Not in this case since it's using `withScope` which is short for:


```
Sentry.pushScope();
callback();
Sentrt.popScope();

```

So the scope within the callback is an isolated clone and changes to it will not be visible outside of the closure.

Reply:
ah sorry I read configureScope somehow